app/models/study.rb

```diff
@@ -1034,7 +1034,11 @@ def has_image_files?
   # into existing endpoints, or perhaps a new endpoint, where the token is returned as part
   # of the API response.
   def has_streamable_files(user)
-    has_bam_files? || has_analysis_outputs?('infercnv', 'ideogram.js') || has_image_files? || user && user.feature_flag_for('differential_expression_frontend')
+    has_bam_files? ||
+    has_analysis_outputs?('infercnv', 'ideogram.js') ||
+    has_image_files? ||
+    user && user.feature_flag_for('differential_expression_frontend') ||
+    self.feature_flag_for('differential_expression_frontend')
```

Comment:
Non-blocking: I know I was a prime offender for this initially in favor of clarity, but `self` is redundant here since this is an instance method.

Reply:
I've added this refinement as an AC in SCP-4375.